README.md

```diff
@@ -1,13 +1,9 @@
 # Project Happy Thoughts API
-
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+The aim of this project was to build and API with both GET request endpoints and POST request endpoint connected to database, as well as deal with validation and error handling. This project is linked to a former project, project-happy-thoughts.
 
 ## The problem
-
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+The project went quite smoothly overall. I was able to follow the instructional materials and build the necessary endpoints/requests from there. I had some problems with deploying on google cloud, but those seem to be resolved, at least for now.
+Tools: MongoDB, Mongoose, Express, MongoDB Compass, MongoDB Atlas, Google Cloud.
```

Comment:
Well done solving the project with all the tools needed - I still struggle with the understanding of it all. I don't really get what's the difference between MongoDB Compass and the Atlas version... If you have time - could you make an effort to explain it to me? 

I guess/think that Atlas is a place for the database to be stored other then on my computer (localhost) but I'm not really sure how to fix it and exactly how to use it...